packages/sui-types/src/lib.rs

```diff
@@ -32,13 +39,19 @@ impl FromStr for SuiAddress {
     type Err = Report<Error>;
 
     fn from_str(s: &str) -> Result<Self, Error> {
```

Comment:
This means that we'll enforce lower case for both source address and destination address. While it's not strictly needed for destination, I prefer the consistency and it's straightforward to adhere to it

Reply:
I'm not sure what your suggestion is

Reply:
Sorry, no suggestion, just a callout that we're restricting Sui destination addresses more than before. I think that's fine for Sui. We don't do the same for EVM though (restrict EVM destination address to be EIP 155 checksum encoded since it's hard to encode to that on-chain)

Reply:
Ya it's better to have the consistency througout amplifier.